components/Footer.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from "styled-components/native";
+import { View, Text, Linking } from 'react-native';
+
+
+export const Footer = () => {
+    return (
+        <FooterContainer>
+            <FooterText>To make someone else's day, please click the link below and write about somehing that made you smile today</FooterText>
+            <FooterLink onPress={() => Linking.openURL('https://spring2022-happy-tweets.netlify.app/')}>Happy Tweets</FooterLink>
+        </FooterContainer>
+    );
+};
+
+
+//----------------------------------------------//
+
+const FooterContainer = styled.View`
```

Comment:
I would say this is easier and much more comfortable working with styling rather than using stylesheet. If you want to test yourself, I would encourage you to do so. You're a great learner so with a little more time you can do it. :D